PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue

```diff
@@ -157,29 +143,29 @@ import EventBus from "../eventBus";
 export default {
   data() {
     return {
-      genderID: ["Male", "Female", "Other", "Decline to answer"],
-      race: [
+      genderIDOptions: ["Male", "Female", "Other", "Decline to answer"],
```

Comment:
Do we typically write this as `genderID` or does it change throughout our code base? I think `genderIdOptions` is better.

Reply:
Changed all references (in all 3 files) to genderIdOptions